lib/custom_wizard/realtime_validations/similar_topics.rb

```diff
@@ -31,7 +32,12 @@ def perform(params)
 
     topics = Topic.similar_to(title, raw, user).to_a
     topics.select! { |t| categories.include?(t.category.id.to_s) } if categories.present?
-    topics.select! { |t| t.created_at > DateTime.parse(date_after) } if date_after.present?
+
+    if time_n_value.present? and time_unit.present?
+      if time_n_value.to_i > 0
```

Comment:
@fzngagan Please combine these into a single conditional and also let's use the standard of ``&&`` instead of ``and`` (unless discourse/discourse is now using that internally?)

Reply:
Ok, I've simplified the condition. I leveraged the fact that `to_i` gives 0 even when its called with nil :)